From fa4f31da87fa4d3537040bb26cfe2f6c82d59e93 Mon Sep 17 00:00:00 2001
From: Stefan Chulski <stefanc@marvell.com>
Date: Thu, 1 Jun 2017 14:42:04 +0300
Subject: [PATCH 1646/2241] fix: net: phy: marvell: Use bitmap for different
 PHY bases

Patch change PHY base indication to bitmap instead of numbers.
Using of numbers could lead to wrong link PHY resolution.

Change-Id: Iace5a9617369651a8d5a4750036f96f1f3490612
Signed-off-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/40094
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 drivers/net/phy/marvell.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/net/phy/marvell.c b/drivers/net/phy/marvell.c
index 6b624b4..e5e217c 100644
--- a/drivers/net/phy/marvell.c
+++ b/drivers/net/phy/marvell.c
@@ -83,9 +83,9 @@
 #define MII_M1111_HWCFG_FIBER_COPPER_AUTO	0x8000
 #define MII_M1111_HWCFG_FIBER_COPPER_RES	0x2000
 
-#define COPPER_BASE_T		1
-#define FIBER_BASE_R		2
-#define FIBER_BASE_X		3
+#define COPPER_BASE_T		BIT(1)
+#define FIBER_BASE_R		BIT(2)
+#define FIBER_BASE_X		BIT(3)
 
 #define MII_M1112_PHY_ENERGY_STATE	0x10
 
-- 
2.7.4

